README.md

```diff
@@ -44,30 +58,9 @@ parameters of the run (
     where to look for other config files
 ).
 
-# Helpful notebooks
-
-- [notebooks/sgf-explorer.ipynb](notebooks/sgf-explorer.ipynb) loads self/victim-play games into a pandas dataframe and lets you do some data analysis. One thing I use this file for is to pick out specific games I then load into a visualizer.
-
-# Prerequisite & Dependencies
-
-- **Summary**: For this project, we use **Docker** and **Git** to set up the dependencies and environment.
-- **Docker image**: HumanCompatibleAI/go_attack:latest
-- **GitHub repo**:
-    - Project code: https://github.com/HumanCompatibleAI/go_attack
-    - Agent code: https://github.com/HumanCompatibleAI/KataGo-custom
-    - Controller code: https://github.com/HumanCompatibleAI/gogui
-- **Setting up**
-    - Download the repo
-        - `git clone --recurse-submodules https://github.com/HumanCompatibleAI/go_attack.git`
-        - `cd go_attack`
-    - Build C++ and Python Docker containers, and run the C++ container. A KataGo executable can be found in the `/engines/KataGo-custom/cpp` directory inside the container.
-        - `docker build . -f compose/cpp/Dockerfile -t humancompatibleai/goattack:cpp`
-        - `docker build . -f compose/python/Dockerfile -t humancompatibleai/goattack:python`
-        - `docker run --gpus all -v ~/go_attack:/go_attack -it humancompatibleai/goattack:cpp`
```

Comment:
I think this `docker run` command and the statement "A KataGo executable can be found in the `/engines/KataGo-custom/cpp` directory inside the [C++] container" are still useful.

Either move the useful information here to the `## Individual containers` section, or add it inside the header comment of the C++ Dockerfile?

Reply:
Added back in.